courses-app/src/app/features/courses/course-list/course-list.component.html

```diff
@@ -7,3 +7,8 @@
     </ng-container>
   </ng-container>
 </app-course-card>
+<p (click)="demoNgrx()" class="btn btn-success m-3">Click me - trigger Ngrx REQUEST_CURRENT_USER_START</p>
+<p> Loading? - {{ userState.$isLoading | async }}</p>
```

Comment:
as a bonus, read about https://ngrx.io/guide/component/push
It removes some caveats of using async pipe